### ISSUE TITLE
Accessibility: screen reader not reading options properly

### ISSUE THREAD

**Issue by BillyTheKidd88** [shentao/vue-multiselect#1009]
### Reproduction Link
https://jsfiddle.net/shentao/jqofkzxc/3296/

### Steps to reproduce
1) Run a the NVDA screen reader.
2) Click on the vue-multiselect
3) Use arrow keys to navigate options

### Expected behaviour
Screen readers reads the option that is highlighted 

### Actual behaviour
Screen readers reads "blank" not the option that is highlighted

**Comment by lenzls** [comment]
just wanted to say, that I tried out @BillyTheKidd88's fix and it seems to work fine.

Glad that you picked this up!

**Comment by SampsonJW** [comment]
> BillyTheKidd88

I can't seem to find the fix you're referring to. Do you mind helping me out? :)

**Comment by pat-norris** [comment]
@SampsonJW 

Is the link to the pull request I put in the issue not working for you? Here it is again: https://github.com/shentao/vue-multiselect/pull/1007
Or are you looking for the code changes specifically? If so here they are: https://github.com/shentao/vue-multiselect/pull/1007/commits/ffb39fb5a094528982db22e7d759ab8a3c7a04ea

**Comment by SampsonJW** [comment]
@pat-norris 

I am having the "Blank" issue using NVDA. From what I understand this should be fixed with your PR?
Or is there something manual I should be doing to fix it?

Thanks for responding :)

**Comment by pat-norris** [comment]
@SampsonJW 

Strange. The pull request was merged to master so there shouldn't be anything you need to do manually. Maybe make sure that the version you have for vue multiselect is up to date. I forget what version number this change was added in but if you get the latest you should be good. If that doesn't work then I'm not really sure.

**Comment by pat-norris** [comment]
@SampsonJW 

Yeah I see that now. Looks like it was only added to 3.0.0-alpha-1.

**Comment by JohnnyChenJiaHao** [comment]
Do you know when v3 will be released?
